generators/app/index.js

```diff
@@ -5,6 +5,25 @@ var sort = require('sort-object');
 var mergeAndConcat = require('merge-and-concat');
 var travisConfigKeys = require('travis-config-keys');
 var ramda = require('ramda');
+var got = require('got');
+
+var supportedVersions = got('https://nodejs.org/dist/index.json', {
```

Comment:
We should handle the failure case too; if it fails, we just ignore and defaults to a simple subset? Like only `- node`

Reply:
Wouldn’t `- lts/*` make more sense? We could also guess the currently supported versions through the release schedule.

Reply:
Yeah, I just think this is an edge case, so we should do the simplest thing with lowest maintenance cost

Reply:
I’ve tried this locally (by disabling the internet connection) and it’s working, but I’m not really sure how to add tests for this.